pkg/commands/app.go

```diff
@@ -884,6 +884,7 @@ func NewVMCommand(globalFlags *flag.GlobalFlagGroup) *cobra.Command {
 		ScanFlagGroup:          flag.NewScanFlagGroup(),
 		SecretFlagGroup:        flag.NewSecretFlagGroup(),
 		VulnerabilityFlagGroup: flag.NewVulnerabilityFlagGroup(),
+		VMFlagGroups:           flag.NewVMFlagGroup(),
```

Comment:
You can define a custom flag as below.

```
awsFlagGroup := flag.AWSFlagGroup{
	Region: flag.Flag{
		Name:       "aws-region",
		ConfigName: "aws.region",
		Value:      "",
		Usage:      "AWS region to scan",
	}
}
```

The following variable should be exported so it could be reused. I'll refactor that later.
https://github.com/aquasecurity/trivy/blob/f369bd3e3dbc3f64135e361bca8059ecd4113e91/pkg/flag/aws_flags.go#L4-L9